packages/create-expo-app/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "create-expo-app",
+  "version": "2.1.1",
+  "bin": "./build/index.js",
+  "main": "build",
+  "description": "Create universal Expo apps",
+  "license": "BSD-3-Clause",
+  "keywords": [
+    "expo",
+    "react-native",
+    "react"
+  ],
+  "homepage": "https://docs.expo.dev",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/expo/expo.git",
+    "directory": "packages/create-expo-app"
```

Comment:
Make the folder be `create-expo` and move stuff accordingly. `bun create expo` > `bun create expo-app`.